### DISCUSSION TITLE
ordered list nested counters not working

### DISCUSSION THREAD

**Discussion by heather-lau** [tailwindlabs/tailwindcss#7032] [Help]
I want to use nested counters but it doesn't support
```
<ol className='list-decimal'>
  <li>Yellow</li>
  <li>Blue
    <ol>
      <li>Red</li>
      <li>Green</li>
    </ol>
  </li>
</ol>
```
I excepted 1, 2, 2.1, 2.2, thanks

**Comment by wongjn** [comment]
This is not really a matter of support, since default [`<ol>` nesting does not do nested numbering either](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol#nesting_lists).

You'll want to use [CSS counters](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters) and MDN helpfully have an [example of nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters#example_of_a_nested_counter) that you could emulate with Tailwind:

```html
<ol class="pl-4 [counter-reset:section]">
  <li class="[counter-increment:section] marker:[content:counters(section,'.')]">Yellow</li>
  <li class="[counter-increment:section] marker:[content:counters(section,'.')]">Blue
    <ol class="pl-4 [counter-reset:section]">
      <li class="[counter-increment:section] marker:[content:counters(section,'.')]">Red</li>
      <li class="[counter-increment:section] marker:[content:counters(section,'.')]">Green</li>
    </ol>
  </li>
</ol>
```
[Here's a Tailwind Play of it in action](https://play.tailwindcss.com/A8rvM7li4s)

**Reply by Serenit8** [reply]
I appreciate the information and advice you have shared. I will try to figure it out for more.
[MyPaymentsPlus App](https://www.mypayments.plus/)

**Reply by Spudisis** [reply]
Doesn't work in Safari

**Reply by NKapali** [reply]
Safari and Firefox still seems to have some issues.
